src/widget.cpp

```diff
@@ -1383,7 +1423,11 @@ std::string widget::layout( const avatar &ava, const unsigned int max_width, int
             // Stack rows vertically into a multiline widget
             for( const widget_id &wid : _widgets ) {
                 widget cur_child = wid.obj();
-                ret += sep + cur_child.layout( ava, max_width, label_width );
+                if( cur_child._separator != default_separator && cur_child._separator != _separator ) {
+                    ret += sep + cur_child.layout( ava, max_width, label_width );
+                } else {
+                    ret += sep + cur_child.layout( ava, max_width, label_width );
+                }
```

Comment:
```suggestion
                ret += sep + cur_child.layout( ava, max_width, label_width );
```
You don't need 2 branches that do the same thing here